### ISSUE TITLE
preserve-symlinks proxy file incorrect location

### ISSUE THREAD

**Issue by vjpr** [rstacruz/pnpm#443]
I am having an issue the preserve-symlink proxy file.

When using create-react-app:

```
Error: Cannot find module '../react-scripts/bin/react-scripts.js'
node_modules/react-scripts/bin/react-scripts.js
```

```
#!/bin/sh
":" //# comment; exec /usr/bin/env node --preserve-symlinks "$0" "$@"
require('../react-scripts/bin/react-scripts.js')
```

It should be: https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/bin/react-scripts.js

This looks like a bug. The bin dir shouldn't be touched, only the top-level `.bin`.
### Code to reproduce the issue:

```
create-react-app
npm start
```

### Expected behavior:

No errors when visiting localhost:3000.

### Actual behavior:

```
Error: Cannot find module '../react-scripts/bin/react-scripts.js'
```

### Additional information:

 - `node -v` prints: 6.5.0
 - Windows, OS X, or Linux?: macOS

**Comment by vjpr** [comment]
When I run `node --preserve-symlinks node_modules/react-scripts/bin/react-scripts.js start` it doesn't give me that error. But returns other errors about not being able to find react.

```
Error in /Users/Vaughan/.pnpm/.store/nested/react-dom@15.3.2/_/index.js
Module not found: 'react/lib/ReactDOM' in /Users/Vaughan/.pnpm/.store/nested/react-dom@15.3.2/_

 @ /Users/Vaughan/.pnpm/.store/nested/react-dom@15.3.2/_/index.js 3:17-46
```

Would pnpm global store cause problems here?

**Comment by zkochan** [comment]
I did not have your first issue on Linux, so it is probably macOS specific.

I did have the second issue though, and I think I have something similar in an angular app ([here](https://github.com/rstacruz/pnpm/issues/403#issuecomment-255882248)).

From this webpack issue it seems as if it can work with the `--preserve-symlinks` option https://github.com/webpack/webpack/issues/2937

Maybe some program spawn's webpack without the `--preserve-symlinks` option?
